### DISCUSSION TITLE
Add sort-by control in Albums > All view

### DISCUSSION THREAD

**Discussion by emdash-ie** [navidrome/navidrome#2846] [Ideas]
As I understand it, the only way to sort the Albums > All view at the moment is to edit the sort-by query parameter in the url.

This is slightly less convenient than a UI control, and makes it hard to tell what values the field could take – I guess it’s probably database columns?

This request is for a dedicated UI control (e.g. a drop-down or search field) to choose the sort order. Ideally I would also like the sort order to remain the same on subsequent visits to the page, until it is changed.

**Comment by deluan** [comment]
Thanks for the suggestion. Yes, this will be an option in the new UI we are working on. No promises on dates, though :)

**Reply by emdash-ie** [reply]
Ok, great! I might have a look at adding it to the current UI while I wait

**Comment by ghost** [comment]
Well, my situation seems at the moment. It seems that the server is currently sorting by the first letter of the album in reverse. Albums starting with a are on the last page of the album list.

**Reply by metalheim** [reply]
You can (a bit annoying, but possible!) sort albums by:

1. switch from Grid view to table view
2. Sort by an available colum
3. Switch back to grid view

**Reply by ghost** [reply]
Oh,yes.Thank you,it works.
